internal/parsers/markdown.go

```diff
@@ -74,6 +75,7 @@ func ExtractScenarioTitleFromAst(node ast.Node, source []byte) (string, error) {
 }
 
 var expectedSimilarityRegex = regexp.MustCompile(`<!--\s*expected_similarity=\s*(\d+\.?\d*)\s*-->`)
+var expectedContainsRegex   = regexp.MustCompile(`<!--\s*expected_contains="(.*)"\s*-->`)
```

Comment:
Shouldn't it "escape" also whitespace character at the beginning as done in `expected_similarity`: `\s*"(.*)"\s*`?